public/scripts/financeController.js

```diff
@@ -0,0 +1,57 @@
+(function(module) {
+
+  Filing = {};
+  Filing.totalPerQuarter = [];
+  Filing.all = [];
+
+  Contributor = {};
+  Contributor.contributors = [];
```

Comment:
Since you're explicitly creating the object literal anyways, you could just add the property right inside the object and simplify a little... it's a very minor thought.

``` javascript 
Contributor = {
  contributors: [],
};
```